src/OSPSuite.R/Domain/AgingData.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Data;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.R.Domain
+{
+   public class AgingData
```

Comment:
@Yuri05 It's very hard to create DataTable and have them usable in R
So I created this very stupid class that will need to be instantiated in R (via a wrapper)